en/references/wireless_access_guides.md

```diff
@@ -32,3 +32,6 @@
   * *Guide:* ["Cracking WPA/WPA2 with oclHashcat"](http://hashcat.net/wiki/doku.php?id=cracking_wpawpa2) (Hashcat wiki)
 
   * *Guide:* ["Wireless Network Review"](http://www.amanhardikar.com/mindmaps/Wireless.html) (amanhardikar.com)
+
+  * *Guide:* ["Router HAcking"](http://www.amanhardikar.com/mindmaps/Wireless.html) (https://github.com/justinbeatz)
```

Comment:
@hackatom I see the same link twice (http://www.amanhardikar.com/mindmaps/Wireless.html)

Reply:
And I think the Justinbeatz link probably needs to point to a specific repo, I'm guessing https://github.com/justinbeatz/Armory ?

Reply:
or, https://github.com/justinbeatz/Stockpile